x-pack/docs/en/rest-api/license/start-trial.asciidoc

```diff
@@ -40,7 +40,7 @@ The following example checks whether you are eligible to start a trial:
 
 [source,js]
 ------------------------------------------------------------
-POST _xpack/license/start_trial
+GET _xpack/license/start_trial
```

Comment:
I think we should make `start_trial` start with an underscore like other APIs that perform an action (`_search`, `_authenticate`, etc), so `_start_trial` and `_start_basic`

Reply:
https://github.com/elastic/x-pack-elasticsearch/issues/1941#issuecomment-322642475

That was discussed. Also I imagine that this "bug fix" PR to add necessary ack messages to a response would not be the place to modify a route that already exists in prior versions of ES x-pack.